packages/backend/src/plugins/scaffolder.ts

```diff
@@ -111,6 +116,32 @@ export default async function createPlugin({
     }
   }
 
+  const azureConfig = config.getOptionalConfig('scaffolder.azure');
+  if (azureConfig) {
+    try {
+      const baseUrl = azureConfig.getString('baseUrl');
+      const azureToken = azureConfig.getConfig('api').getString('token');
+
+      const authHandler = getPersonalAccessTokenHandler(azureToken);
+      const webApi = new WebApi(baseUrl, authHandler);
+      const azureClient = await webApi.getGitApi();
+
+      const azurePublisher = new AzurePublisher(azureClient, azureToken);
+      publishers.register('azure/api', azurePublisher);
```

Comment:
I wonder if we can just `azure` instead of `azure/api`

Reply:
I thought this was connected to the 'azure/api' type in the catalog, when a location gets ingested. If it's not then I'm happy to change it.

Reply:
It's all connected, this diversion came when we introduced the `github/api` version which would use an authenticated github client instead, but not sure that this needs to be divided. As it's always going to be authenticated, and i'd really like to get rid of the different `github` and `github/api` versions and just have one.

Thoughts @freben @Rugvip?

Reply:
Yeah, we should definitely merge them all into one per provider, unless there is particularly good reason to break something out.

Reply:
How would you like me to proceed with this part? I'm still hoping to get this in to a release soon to be able to start using a stand-alone app and not build from the main repo.

Reply:
@benjdlambert or @freben do I need to do some changes here? It feels like getting rid of the `/api` part could be separate issue.

Reply:
I think let's leave it for now, but we should clear them up at a later date.